varlink/scanner.py

```diff
@@ -291,7 +291,7 @@ def filter_params(self, parent_name, varlink_type, _namespaced, args, kwargs):
                     )
                 return args
             else:
-                InvalidParameter(parent_name)
+                return InvalidParameter(parent_name)
```

Comment:
Further down below `InvalidParameters` is raised on line 344. Can you explain where you see that InvalidParameter is returned, I might be missing something.